### ISSUE TITLE
Tabs don't show correct activities

### ISSUE THREAD

**Issue by LEDfan** [nextcloud/activity#283]
### Steps to reproduce
1. Updated Nextcloud to 14
2. Load the activity app
3. Click on "Favorites"

### Expected behaviour
The app should load the activities related to files which are marked as favorite.

### Actual behaviour
Activities which are done by others are shown.
All the tabs gives weird results and I'm also not sure if it's always the same.

### Server configuration
**Operating system**: CentOS 7.5

**Web server:** Apache 2.4.6

**Database:** MySQL 5.5.56

**PHP version:** 7.2.8

**Nextcloud version:** 14.0.0 beta1 (14.0.0.13)

**Where did you install Nextcloud from:** zip (updated from NC 13.0.5)

**Signing status:**

```
No errors have been found.
```

**List of activated apps:**

```
  - accessibility: 1.0.1
  - activity: 2.7.0
  - calendar: 1.6.1
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - contacts: 2.1.5
  - dav: 1.6.0
  - deck: 0.4.0
  - drawio: 0.8.9
  - end_to_end_encryption: 1.0.5
  - federatedfilesharing: 1.4.0
  - federation: 1.4.0
  - files: 1.9.0
  - files_external: 1.5.0
  - files_pdfviewer: 1.3.2
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - gallery: 18.1.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - mail: 0.8.3
  - news: 12.0.4
  - nextcloud_announcements: 1.3.0
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - password_policy: 1.4.0
  - provisioning_api: 1.4.0
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - support: 1.0.0
  - survey_client: 1.2.0
  - systemtags: 1.4.0
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.0
  - updatenotification: 1.4.1
  - workflowengine: 1.4.0
```

**Nextcloud configuration:**

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "14.0.0.13",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "theme": "",
        "maintenance": false,
        "maxZipInputSize": 0,
        "allowZipDownload": true,
        "secret": "***REMOVED SENSITIVE VALUE***",
        "app.mail.server-side-cache.enabled": true,
        "trashbin_retention_obligation": "auto",
        "preview_max_scale_factor": 1,
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "enabledPreviewProviders": {
            "0": "OC\\Preview\\PNG",
            "1": "OC\\Preview\\JPEG",
            "2": "OC\\Preview\\GIF",
            "11": "OC\\Preview\\Illustrator",
            "12": "OC\\Preview\\Postscript",
            "13": "OC\\Preview\\Photoshop",
            "14": "OC\\Preview\\TIFF"
        },
        "htaccess.RewriteBase": "\/",
        "overwrite.cli.url": "",
        "loglevel": 2,
        "updater.release.channel": "beta",
        "mysql.utf8mb4": true,
        "mail_smtpmode": "php",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***"
    }
}
```

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:** FF 60

**Operating system:** openSUSE leap 15
 
#### Nextcloud log (data/nextcloud.log)
```
Nothing related
```

#### Browser log
```
Nothing related
```

I also noticed using the network tab of the developer tools that the app is sending a lot of requests:
![selectie_0024](https://user-images.githubusercontent.com/2996275/43302805-04eb87c2-916d-11e8-8864-7e4fcf8d40f0.png)

**Comment by nickvergessen** [comment]
For me each tab requested the correct filter URL.
The screenshot you show seems to only request "by" which is "By others"?

**Comment by nickvergessen** [comment]
@skjnldsv can you help here?

The problem is the scroll container size is wrong I guess:
https://github.com/nextcloud/activity/blob/35805e766f58c449023439c95e0b973580d948cb/js/script.js#L52-L54

**Comment by LEDfan** [comment]
I can confirm it's fixed by #293
